update.sh

```diff
@@ -100,25 +105,24 @@ echo '#### DO NOT MODIFY. THIS FILE IS AUTOGENERATED ####
 ' | cat - travis.yml.template >.travis.yml
 
 for version in "${versions[@]}"; do
-  # Skip "docs" and other non-docker directories
-  [ -f "${version}/Dockerfile" ] || continue
+  parentpath=$(dirname "${version}")
+  versionnum=$(basename "${version}")
+  baseuri=$(get_config "${parentpath}" "baseuri")
+  update=$(in_versions_to_update "${version}")
 
-  parentpath=$(dirname "$version")
-  versionnum=$(basename "$version")
-  baseuri=$(get_config "$parentpath" "baseuri")
-  update=$(in_versions_to_update "$version")
+  # Get supported variants according the target architecture
+  # See details in function.sh
+  IFS=' ' read -ra variants <<<"$(get_variants "${parentpath}")"
 
-  add_stage "${baseuri}" "${version}" "default"
+  if [ -f "${version}/Dockerfile" ]; then
+    add_stage "${baseuri}" "${version}" "default"
 
-  if [ "${update}" -eq 0 ]; then
-    info "Updating version $version..."
-    update_node_version "${baseuri}" "${versionnum}" "${parentpath}/Dockerfile.template" "${version}/Dockerfile" &
+    if [ "${update}" -eq 0 ]; then
+      info "Updating version ${version}..."
```

Comment:
Same here.

Reply:
This issue is not so straight in normal diff but in the diff mode that hides whitespace changes, so it depends.